library/tiqr/tests/Tiqr_UserSecretStorageTest.php

```diff
@@ -5,16 +5,45 @@
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
 
+class CustomEncryptionClass_2 implements Tiqr_UserSecretStorage_Encryption_Interface
```

Comment:
Rule of thumb is to have 1 class per file. Given this is a test, it's excusable to add a test class to the existing test. But this gets messy real quickly in my opinion.